web/src/echo.ts

```diff
@@ -153,6 +221,7 @@ export function build_display_recipient(message) {
             id: person.user_id,
             email: person.email,
             full_name: person.full_name,
+            is_mirror_dummy: false,
```

Comment:
Can we drop `is_mirror_dummy` from the type? I don't see it being used anywhere.
Am I missing something?

Reply:
Done in https://github.com/zulip/zulip/pull/30702/commits/92a40d80a83c3ac8208d50218d3f23ceac190b8d

Reply:
We do have upcoming features that might use it, so it is probably better not to drop it.